app/models/ip_address_matcher.rb

```diff
@@ -18,6 +19,6 @@ def cidrs(terms)
   end
 
   def cidr(term)
-    NetAddr::CIDR.create(term)
+    NetAddr.parse_net(term)
```

Comment:
Upgrading from NetAddr 1.x to 2.x involved some changes to the app code, as the new version isn't backwards compatible with the old.